providers/go-feature-flag/src/main/java/dev/openfeature/contrib/providers/gofeatureflag/GoFeatureFlagProvider.java

```diff
@@ -215,8 +218,7 @@ private <T> ProviderEvaluation<T> resolveEvaluationGoFeatureFlagProxy(
                 }
 
                 return ProviderEvaluation.<T>builder()
-                        .errorCode(goffResp.getErrorCode())
```

Comment:
I removed the error code here. I think that if we got this far, we don't need the error? Otherwise we should probably throw.

I think this is the biggest change I made.